pip/pep425tags.py

```diff
@@ -164,23 +164,39 @@ def get_supported(versions=None, noarch=False):
             # support macosx-10.6-intel on macosx-10.9-x86_64
             match = _osx_arch_pat.match(arch)
             if match:
```

Comment:
With what I had in mind, the input would be the content of `'macosx_{0}_{1}_{2}'.format(major, minor, machine)` and the ouput would be the content of `arches`.